.travis.yml

```diff
@@ -1,23 +1,16 @@
 language: node_js
-node_js:
-  - lts/*
-cache:
-  directories:
-    - "~/.npm"
+node_js: 12
 before_install:
-  - npm install -g npm@latest
-  - npm install -g greenkeeper-lockfile@1.14.0
+- npm install -g npm@6
```

Comment:
This is unneeded, you'll get npm 6.11 with node 12. The whole before_install block can go away I think.

Reply:
Oh, nice.